{
    "compiled": false,
    "function_name": "1_fpr_lt",
    "variant_name": "liboqs",
    "project": "liboqs",
    "python_error_trace": "Traceback (most recent call last):\n  File \"/home/javier/Galapagos/scripts/tests/usecase.py\", line 49, in compare\n    await variant.compile(variant_shadow) # raises exception on fail\n  File \"/home/javier/Galapagos/scripts/usecases/case.py\", line 554, in compile\n    self.replace_bc(cwd)\n  File \"/home/javier/Galapagos/scripts/usecases/case.py\", line 449, in replace_bc\n    subprocess.check_output([\n  File \"/usr/lib/python3.10/subprocess.py\", line 421, in check_output\n    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,\n  File \"/usr/lib/python3.10/subprocess.py\", line 526, in run\n    raise CalledProcessError(retcode, process.args,\nsubprocess.CalledProcessError: Command '['opt', '-strip-debug', '/home/javier/Galapagos/functions/liboqs/variants/c/1_fpr_lt_8.bc', '-o', '/home/javier/Galapagos/functions/liboqs/variants/c/1_fpr_lt_8.bc']' returned non-zero exit status 1.\n",
    "python_error_msg": "Command '['opt', '-strip-debug', '/home/javier/Galapagos/functions/liboqs/variants/c/1_fpr_lt_8.bc', '-o', '/home/javier/Galapagos/functions/liboqs/variants/c/1_fpr_lt_8.bc']' returned non-zero exit status 1."
}